ledger-api/grpc-definitions/com/daml/ledger/api/v1/expected_ledger_feature_descriptors.yaml

```diff
@@ -0,0 +1,58 @@
+# Example feature descriptor for our existing ledgers.
+# Written as pseudo YAML using its mapping to JSON and then to `FeatureDescriptor` proto message
+
+sandbox-classic:
+  invocation_matrix:
+    # TODO
+  ledger:
+    ordered_contract_ids: 
+      # set only if contract-id seeding is not weak
+    monotonic_record_time:
+  participant:
+    party_allocation:
+      implicit_party_allocation: True 
+    participant_pruning:
+      supports_pruning_all_divulged_events: False
+    ledger_config_management:
+  experimental:
+    rank_based_command_deduplication:
+      completions_include_submission_ids: True
+
+sandbox-next:
+  invocation_matrix:
```

Comment:
What is the intention behind this invocation matrix? It doesn't show up in the Protobuf definitions and as plain text strings I don't see how to automatically generate invocations for this.